java-bootcamp-bahia-t1/src/Transaction.java

```diff
@@ -0,0 +1,37 @@
+/* Represents a transaction :
+ *  0. Receives a full cart from customer.
+ *  1. Chose the form of payment.
+ *  2. Apply the correspondent discount.
+ *  3. Make payment.                             */
+
+public class Transaction {
+
+	private ShoppingCart cart;
+	private static int ID = 0;

```

Comment:
Could you use a pattern to provide unique sequential numbers and use it to set an identification to each payment transaction?